pkg/api/management/api/logging/validator.go

```diff
@@ -0,0 +1,87 @@
+package logging
+
+import (
+	"net"
+	"net/url"
+
+	"github.com/rancher/norman/httperror"
+	"github.com/rancher/norman/types"
+	"github.com/rancher/norman/types/convert"
+	"github.com/rancher/types/apis/management.cattle.io/v3"
+)
+
+func ClusterLoggingValidator(resquest *types.APIContext, schema *types.Schema, data map[string]interface{}) error {
+	found := false
+	if es, ok := data["ElasticsearchConfig"]; ok {
+		found = true
```

Comment:
Nit: could we extract each config validation into its own method: validateElasticsearchConfig/validateSplunkConfig, etc